catalyst/engines/amp.py

```diff
@@ -106,7 +106,7 @@ def pack_checkpoint(
             criterion=criterion,
             optimizer=optimizer,
             scheduler=scheduler,
-            **checkpoint,
+            **kwargs,
```

Comment:
scaler is missing...

Reply:
I see now why there was mistake, but we need to transfer kwargs too. So maybe it will be better to add scaler into kwargs?

Reply:
you could pass `scaler=scaler`, it should work ;)